internal/agent/install.go

```diff
@@ -127,22 +120,18 @@ func Install(dir ...string) error {
 	// runs the installation
```

Comment:
I'm glad to see all this weird juggling of options in this function to be gone now. I just wonder if they were addressing some very specific use case or they are just relics from when we had multiple configs around. I hope it's just the latter.